eng/testing/tests.mobile.targets

```diff
@@ -232,9 +232,22 @@
   <Import Project="$(MonoProjectRoot)\wasm\build\WasmApp.InTree.targets" Condition="'$(TargetOS)' == 'Browser'" />
   <PropertyGroup>
       <WasmBuildAppDependsOn>PrepareForWasmBuildApp;$(WasmBuildAppDependsOn)</WasmBuildAppDependsOn>
+      <EmSdkDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'src', 'mono', 'wasm', 'emsdk'))</EmSdkDir>
   </PropertyGroup>
 
-  <Target Condition="'$(TargetOS)' == 'Browser'" Name="BundleTestWasmApp" DependsOnTargets="WasmBuildApp" />
+  <Target Condition="'$(TargetOS)' == 'Browser'" Name="BundleTestWasmApp" DependsOnTargets="WasmBuildApp;StageEmSdkForHelix" />
+
+  <!-- CI has emscripten provisioned in $(EMSDK_PATH) as `/usr/local/emscripten`. Because helix tasks will attempt to write a .payload file, we cannot use $(EMSDK_PATH) to package emsdk as a helix correlation payload. Instead, we copy over the files to a new directory `src/mono/wasm/emsdk` and use that. -->
```

Comment:
Sorry, I didn't put it on multiple lines
```suggestion
  <!-- CI has emscripten provisioned in $(EMSDK_PATH) as `/usr/local/emscripten`. Because helix tasks will
   attempt to write a .payload file, we cannot use $(EMSDK_PATH) to package emsdk as a helix correlation 
   payload. Instead, we copy over the files to a new directory `src/mono/wasm/emsdk` and use that. -->
```